src/service/topics.ts

```diff
@@ -1,4 +1,5 @@
 import crashlytics from '@react-native-firebase/crashlytics';
+import moment from 'moment';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the `moment` library is a good choice for handling date and time manipulations. However, it's important to ensure that this new dependency doesn't introduce any version conflicts or other issues with existing dependencies in your project.

```diff
+ import moment from 'moment';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->